### PR TITLE
Use default machinepool instead of additional autoscaling one

### PR DESCRIPTION
**Pull request by mhajas**

### CODE REVIEW

**Comment by kami619** [comment]
@mhajas I think the changes look good to me.

File: .github/workflows/keycloak-scalability-benchmark.yml [hunk 1]
```diff
@@ -181,10 +177,6 @@ jobs:
         with:
           project: ${{ env.PROJECT }}
 
-      - name: Scale down the cluster
```
Comment (kami619) [inline .github/workflows/keycloak-scalability-benchmark.yml:184]: Would we still want this to be there? If the cluster decides to autoscale based on this scaling group?

Comment (mhajas) [reply .github/workflows/keycloak-scalability-benchmark.yml:184]: This step is not scaling down anything if the resources are still used. It is allowing the machinepool to scale to 0 if nodes are not needed. After this PR we are never setting min-replicas to > 0 so we don't need this step.

**Comment by ryanemerson** [comment]
@mhajas Do you have a link to a successful action creating clusters?

**Comment by mhajas** [comment]
> @mhajas Do you have a link to a successful action creating clusters?

I don't have any. My idea was to test it in "production" 😈. I can give it a try though.

**Comment by ryanemerson** [comment]
After discussing on slack, we have agree to merge and fix any issues that appear on the daily run tomorrow.

**Pull request merged**